composer.json

```
@@ -38,7 +38,7 @@
     "require-dev": {
         "behat/behat":           "^3.14",
         "symfony/filesystem":    "^5.4  || ^6.0 || ^7.0",
-        "phpunit/phpunit":       "^9.0 || ^10.0",
+        "phpunit/phpunit":       "^9.0 || ^10.0 || ^11.0",
         "vimeo/psalm":           "^4.3 || ^5.2",
```

Allowing Psalm 6.0 will allow PHPUnit 11 installation.

```suggestion
        "vimeo/psalm":           "^4.3 || ^5.2 || ^6.0",
```

---

I've done #1484 for this exact purpose.